static/admin.js

```diff
@@ -17,11 +17,15 @@ websocket.onopen = (_event) => {
 
 websocket.onmessage = (event) => {
     let data = JSON.parse(event.data)
-    console.log(data)
-    if (data['topic'] === 'counts') {
-        setData(data.data);
-    } else if (data['topic'] === 'message') {
-        setMessage(data.data);
+    if (!('ping' in data)) {
+        console.log(data)
+        if (data['topic'] === 'counts') {
+            setData(data.data);
+        } else if (data['topic'] === 'message') {
+            setMessage(data.data);
+        } else if (data['topic'] === 'frozen') {
+            setFrozen(data.data)
+        }
```

Comment:
Kris would say to avoid the if else stack by using a dictionary with functions in its values

Reply:
I know like I did with the admin messages, but this is the last one I swear

Reply:
That's what they all say